cmd/containerd-shim-runhcs-v1/pod.go

```diff
@@ -135,7 +136,28 @@ func createPod(ctx context.Context, events publisher, req *task.CreateTaskReques
 		host:   parent,
 	}
 	if oci.IsWCOW(s) {
-		// For WCOW we fake out the init task since we dont need it.
+		// For WCOW we fake out the init task since we dont need it. We only
+		// need to provision the guest network namespace.
+		nsid := ""
+		if s.Windows != nil && s.Windows.Network != nil {
+			nsid = s.Windows.Network.NetworkNamespace
+		}
+
+		// TODO: JTERRY75 - Do we need to support other than RS5 hosts here?
```

Comment:
My understanding was that we only use the namespace ID on RS5+ @madhanrm is that correct?

Reply:
I know but if we allow downlevel POD's I am just hinting to myself that we might need to support the EndpointList style workflows.

Reply:
Nvm. We already validate that this impl of the `pod` interface is RS5+. We will write another one to handle downlevel if we need to support things like `endpoint list` etc.